mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -61,7 +61,7 @@ CLASS net/minecraft/unmapped/C_jxtrubuh net/minecraft/enchantment/Enchantment
 		COMMENT {@return gets key for translating the enchant in the lang files}
 	METHOD m_zfqdifee getName (I)Lnet/minecraft/unmapped/C_rdaqiwdt;
 		COMMENT {@return a translation key that is formatted to a specific color}
-		COMMENT By default it changes enchantments that {@return true} in {@code isCursed()} to have red names.
+		COMMENT By default this method changes enchantments that return {@code true} from {@code isCursed()} to have red names.
```

Comment:
Wouldn't linking to `isCursed` be better?

Reply:
maybe. I was mostly fixing issues and this was the only one where I had to change the javadoc a bit